app/controllers/orgs/rosters_controller.rb

```diff
@@ -122,9 +122,25 @@ def add_students
     # rubocop:enable Metrics/AbcSize
 
     def download_roster
+      grouping = Grouping.find_by(id: params[:grouping])
```

Comment:
The linter is warning that this method now has too many lines. Either refactor out some of this grouping logic into a separate method or throw a `# rubocop:disable Metrics/MethodLength` and `# rubocop:enable Metrics/MethodLength` around this method.